web/client/examples/api/localConfig.json

```diff
@@ -1,5 +1,5 @@
 {
-	"proxyUrl": "/mapstore/proxy/?url=",
+	"proxyUrl": "/proxy/?url=",
```

Comment:
I think this will not work on a server. Maybe you should use relative URL (using ..) ? How the other examples do ?

Reply:
@offtherailz I rolled back the change, it seems to be working

Reply:
@offtherailz I looked through localConfig and it sets proxyUrl to relative url, so it works in both instances. The problem with examples is that they are on a separate url /examples/api and proxy is on /proxy so I'm not sure what to do here except adding demo.geo-solutions.it to useCORS so that it doesn't go through the proxy. :man_shrugging: